### PR TITLE
Limit Global Styles: Added the Premium design tools to the signup and plan compare features

### PR DESCRIPTION
**Pull request by rcrdortiz**
#### Proposed Changes

* We now list the `Premium design tools` in the plan comparison screen.

#### Testing Instructions

* Create a new site
* In the plan comparison screen you should see the `Premium design tools` feature for all paid plans.

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Related to 899-gh-Automattic/dotcom-forge

### CODE REVIEW

File: packages/calypso-products/src/constants/features.ts
Comment (mmtr) [inline packages/calypso-products/src/constants/features.ts]: Since this feature is meant to highlight Global Styles with a different marketing name, I think we should still use the existing feature rather than adding a new one:

https://github.com/Automattic/wp-calypso/blob/9a3f6d73f51f951b0cbba9247ff046b41b1ae029/packages/calypso-products/src/constants/features.ts#L269

Comment (rcrdortiz) [reply packages/calypso-products/src/constants/features.ts]: Sure thing.

Comment (rcrdortiz) [reply packages/calypso-products/src/constants/features.ts]: Done.

**Comment by rcrdortiz** [comment]
Changed the position of the `premium design tools` so that it's shown after the `premium design customizations` feature.

File: packages/calypso-products/src/plans-list.tsx
Comment (mmtr) [inline packages/calypso-products/src/plans-list.tsx]: Note that this could use a feature flag check if you want to avoid putting this PR on hold:

```suggestion
			isEnabled( 'limit-global-styles' ) ? WPCOM_FEATURES_GLOBAL_STYLES : null,
```

**Review by mmtr — COMMENTED** [review]
I think the "Premium design tools" term was meant to include and replace other existing features such as "Basic design customization" or "Advanced design customization". Otherwise it might be difficult for users to understand the differences between them.

Note how the design proposal displays just one single combined feature:

image

File: packages/calypso-products/src/plans-list.tsx
Comment (rcrdortiz) [reply packages/calypso-products/src/plans-list.tsx]: I'll leave on hold, that way I don't need to revert anything after.

**Comment by rcrdortiz** [comment]
> I think the "Premium design tools" term was meant to include and replace other existing features such as "Basic design customization" or "Advanced design customization". Otherwise it might be difficult for users to understand the differences between them.
> 
> Note how the design proposal displays just one single combined feature:
> 
> image

In the current calypso screen, there is no reference to any design options so I think your concerns don't apply to this specific case. That is, it's not that they're combined, it's that at the moment we aren't displaying anything regarding design.
Captura de Pantalla 2022-09-27 a las 13 44 53

File: packages/calypso-products/src/plans-list.tsx
Comment (mmtr) [reply packages/calypso-products/src/plans-list.tsx]: I'd strongly recommend to add the feature flag and revert later, so we can ask folks to test everything in the CfT by simply enabling a feature flag.

**Comment by mmtr** [comment]
>  it's that at the moment we aren't displaying anything regarding design.

Sorry, I forgot to mention that this happens in the Upgrades > Plans page:

Screen Shot 2022-09-27 at 13 47 30

**Comment by rcrdortiz** [comment]
> > it's that at the moment we aren't displaying anything regarding design.
> 
> Sorry, I forgot to mention that this happens in the Upgrades > Plans page:
> 
> Screen Shot 2022-09-27 at 13 47 30

I still don't think we should merge them, for me, design tools and design customizations aren't related.

For example, how can we include global styles in both basic and advanced customizations? AFAIK we want to market global styles as a new feature not included in existing features, which our users are already aware of right?

I'll ping @SaxonF so that he can make clear his view on this.

**Comment by mmtr** [comment]
> design tools and design customizations aren't related.

We probably need to find a better copy to don't confuse users because currently it's not clear what's the difference between them:
Screen Shot 2022-09-27 at 13 57 01

Screen Shot 2022-09-27 at 13 57 11

**Comment by rcrdortiz** [comment]
> > design tools and design customizations aren't related.
> 
> We probably need to find a better copy to don't confuse users because currently it's not clear what's the difference between them: Screen Shot 2022-09-27 at 13 57 01
> 
> Screen Shot 2022-09-27 at 13 57 11

Let's get the [Pricing and Packaging team](dgrnI-p2) involved.

**Comment by rcrdortiz** [comment]
[Posted on packaging and pricing P2](pdgrnI-1Ix-p2), let's wait until we get a response before making a decision.

**Comment by mmtr** [comment]
Looks like we finally have a way forward: pdgrnI-1Ix-p2#comment-2914

**Comment by rcrdortiz** [comment]
> Looks like we finally have a way forward: pdgrnI-1Ix-p2#comment-2914

I think we should wait a bit more just in case, have a look at this comment pdgrnI-1Ix-p2#comment-2922 for context.

**Comment by mmtr** [comment]
It's been 4 days without hearing back, so I assume they don't have further feedback. I re-pinged them anyway just in case.

**Comment by rcrdortiz** [comment]
> It's been 4 days without hearing back, so I assume they don't have further feedback. I re-pinged them anyway just in case.

We have a green light now, the person pinged got back to us pdgrnI-1Ix-p2#comment-2929

**Comment by rcrdortiz** [comment]
I've applied the latest agreements we've reached in pdgrnI-1Ix-p2#comment-2929, this is now RfR again.

**Review by dsas — APPROVED** [review]
I think this is _fine_ it does what it's supposed to.

The pricing page (wordpress.com/pricing), isn't affected by this change - indeed I don't think it's even a calypso page and it already uses the phrase "Advanced design customization". 

image


I don't think it's a problem, just bringing it to your attention in case it isn't already. The phrasing & tooltip is a bit confusing, but it doesn't contravene the changes in this PR.

**Comment by rcrdortiz** [comment]
> I think this is _fine_ it does what it's supposed to.
> 
> The pricing page (wordpress.com/pricing), isn't affected by this change - indeed I don't think it's even a calypso page and it already uses the phrase "Advanced design customization".
> 
> image
> 
> I don't think it's a problem, just bringing it to your attention in case it isn't already. The phrasing & tooltip is a bit confusing, but it doesn't contravene the changes in this PR.

Yup, that's not a Calypso page, we have another PR for those changes.

**Review by mmtr — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7568065

Hi @rcrdortiz, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Access extended color schemes and customization tools. Change all of your site's fonts, colours and more.`
- `CSS customization`
- `Add your own CSS code, giving you complete control over how your site looks.`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7568065).

**Pull request merged**
